src/components/Tool/Appeal/Appeals.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { ReactElement } from 'react';
+import {
+  Box,
+  Typography,
+  Divider,
+  CircularProgress,
+  styled,
+} from '@material-ui/core';
+import { useTranslation } from 'react-i18next';
+import { useGetAppealsQuery } from '../../../../pages/accountLists/[accountListId]/tools/GetAppeals.generated';
+import { useAccountListId } from '../../../../src/hooks/useAccountListId';
+import Appeal from '../../../../src/components/Tool/Appeal/Appeal';
+import NoAppeals from '../../../../src/components/Tool/Appeal/NoAppeals';
+
+const LoadingIndicator = styled(CircularProgress)(({ theme }) => ({
+  margin: theme.spacing(0, 1, 0, 0),
+}));
+
+const Appeals = (): ReactElement => {
+  const accountListId = useAccountListId();
+  const { t } = useTranslation();
+  const { data, loading } = useGetAppealsQuery({
+    variables: { accountListId: accountListId || '' },
+  });
+
+  return (
+    <Box mb={1}>
+      <Box m={1}>
+        <Typography variant="h6">{t('Primary Appeal')}</Typography>
+      </Box>
+      <Divider />
+      {loading ? (
+        <Box display="flex" justifyContent="center" mt={10}>
+          <LoadingIndicator color="primary" size={40} />
+        </Box>
+      ) : data?.primaryAppeal && data.primaryAppeal.nodes.length > 0 ? (
+        <>
+          {console.log(data)}
```

Comment:
Still using this?

Reply:
I tried changing it before and was still giving me an error, but I just tried again while removing the uneeded wrapper and now its fine 🤷‍♂️